examples/basic/main.tf

```diff
@@ -0,0 +1,39 @@
+provider "digitalocean" {
+  token = var.do_token
+}
+
+module "terraform_module_digitalocean_vpc" {
```

Comment:
Lets shorten this to `digitalocean_vpc` or even something like `do_vpc` or `example_vpc`.

Reply:
corrected